kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalTopic.java

```diff
@@ -40,9 +43,17 @@ public static InternalTopic from(TopicDescription topicDescription,
                                    List<ConfigEntry> configs,
                                    InternalPartitionsOffsets partitionsOffsets,
                                    Metrics metrics,
-                                   InternalLogDirStats logDirInfo) {
+                                   InternalLogDirStats logDirInfo,
+                                   String internalTopicPrefix) {
```

Comment:
add @Nullable annotation, or do defaulting on higher level (in service)

Reply:
Done, the first way